src/lib/Timeline.js

```diff
@@ -1250,103 +1164,100 @@ export default class ReactCalendarTimeline extends Component {
-            width={width}
-            height={height}
-            onZoom={this.changeZoom}
-            onWheelZoom={this.handleWheelZoom}
-            traditionalZoom={traditionalZoom}
-            onScroll={this.onScroll}
-            isInteractingWithItem={isInteractingWithItem}
-            onMouseEnter={this.handleScrollMouseEnter}
-            onMouseLeave={this.handleScrollMouseLeave}
-            onMouseMove={this.handleScrollMouseMove}
-            onContextMenu={this.handleScrollContextMenu}
+        <TimelineMarkersProvider>
+          <div
+            style={this.props.style}
+            ref={el => (this.container = el)}
+            className="react-calendar-timeline"
           >
-            <div
-              ref={el => (this.canvasComponent = el)}
-              className="rct-canvas"
-              style={canvasComponentStyle}
-            >
-              {this.items(
-                canvasTimeStart,
-                zoom,
-                canvasTimeEnd,
-                canvasWidth,
-                minUnit,
-                dimensionItems,
-                groupHeights,
-                groupTops
-              )}
-              {this.verticalLines(
-                canvasTimeStart,
-                canvasTimeEnd,
-                canvasWidth,
-                minUnit,
-                timeSteps,
-                height,
-                headerHeight
-              )}
-              {this.horizontalLines(canvasWidth, groupHeights)}
-              {this.todayLine(
-                canvasTimeStart,
-                canvasTimeEnd,
-                canvasWidth,
-                height
-              )}
-              {mouseOverCanvas && showCursorLine
-                ? this.cursorLine(
-                    cursorTime,
+            {this.header(
+              canvasTimeStart,
+              zoom,
+              canvasTimeEnd,
+              canvasWidth,
+              minUnit,
+              timeSteps,
+              headerLabelGroupHeight,
+              headerLabelHeight
+            )}
+            <div style={outerComponentStyle} className="rct-outer">
+              {sidebarWidth > 0 ? this.sidebar(height, groupHeights) : null}
+              <ScrollElement
+                scrollRef={el => (this.scrollComponent = el)}
+                width={width}
+                height={height}
+                onZoom={this.changeZoom}
+                onWheelZoom={this.handleWheelZoom}
+                traditionalZoom={traditionalZoom}
+                onScroll={this.onScroll}
+                isInteractingWithItem={isInteractingWithItem}
+                onMouseEnter={this.handleScrollMouseEnter}
+                onMouseLeave={this.handleScrollMouseLeave}
+                onMouseMove={this.handleScrollMouseMove}
+                onContextMenu={this.handleScrollContextMenu}
+              >
+                <div
+                  ref={el => (this.canvasComponent = el)}
+                  className="rct-canvas"
```

Comment:
I see that you removed `.rct-canvas` from scss. Is there a reason why it is there?

Reply:
Nope, will remove!